src/universal-dapp.js

```diff
@@ -58,6 +54,10 @@ UniversalDApp.prototype.reset = function (contracts, getAddress, getValue, getGa
     this._addAccount('d74aa6d18aa79a05f3473dd030a97d3305737cbc8337d940344345c1f6b72eea')
     this._addAccount('71975fbf7fe448e004ac7ae54cad0a383c3906055a65468714156a07385e96ce')
   }
+  this.txRunner = new TxRunner(this.executionContext, this.accounts, {
```

Comment:
hmm so i see in opts just a param used to modify the behavior of txrunner.
executionContext and accounts are more kind of required parameters.